### PR TITLE
issue-14: fixed minor bugs found when testing

### PR DESCRIPTION
**Pull request by IslaL**
Fixed bugs found when testing with new test suites. Descriptions of each bug is in: https://github.com/OceanNetworksCanada/api-matlab-client/issues/14

Bugs found in OncDelivery will be fixed together later with download issue

### CODE REVIEW

File: onc/+onc/DataProductFile.m [hunk 1]
```diff
@@ -101,31 +99,43 @@
                         this.fileSize = strlength(response.Body.Data);
                     end
                     %save_as_file doesn't work properly! Use urlwrite instead
-                    [~, saveResult] = urlwrite(uri.EncodedURI,fullfile(outPath, filename));
-                    %saveResult = util.save_as_file(response, outPath, filename, overwrite);
+                    %[~, saveResult] = urlwrite(uri.EncodedURI,fullfile(outPath, filename));
```
Comment (KristenMeyerONC) [inline onc/+onc/DataProductFile.m]: You can remove this commented code

File: onc/+onc/DataProductFile.m [hunk 2]
```diff
@@ -101,31 +99,43 @@
...
+                        log.printLine(sprintf('Downloaded "%s"\n', this.fileName));
                     elseif saveResult == -2
-                        log.printLine(sprintf('Skipping "%s": File already exists', this.fileName));
+                        log.printLine(sprintf('Skipping "%s": File already exists\n', this.fileName));
                         this.status = 777;
                     end
                 elseif s == 202
                     % Still processing, wait and retry
                     log.printResponse(jsondecode(response.Body.Data));
+                    %log.printResponse(response.Body.Data);
```
Comment (KristenMeyerONC) [inline onc/+onc/DataProductFile.m]: And this commented code

File: onc/+onc/DataProductFile.m [hunk 3]
```diff
@@ -101,31 +99,43 @@
                         this.fileSize = strlength(response.Body.Data);
                     end
                     %save_as_file doesn't work properly! Use urlwrite instead
-                    [~, saveResult] = urlwrite(uri.EncodedURI,fullfile(outPath, filename));
-                    %saveResult = util.save_as_file(response, outPath, filename, overwrite);
+                    %[~, saveResult] = urlwrite(uri.EncodedURI,fullfile(outPath, filename));
+                    % neither urlwrite nor save_as_file works properly, use websave
```
Comment (KristenMeyerONC) [inline onc/+onc/DataProductFile.m]: Can you update this comment to make it in line with the comment on line 101? 

101 says to use urlwrite, then this says not to use urlwrite

**Review by KristenMeyerONC — CHANGES_REQUESTED** [review]

File: onc/+onc/DataProductFile.m (hunk 3 above)
Comment (IslaL) [reply onc/+onc/DataProductFile.m]: Sure! I forgot the commented lines

**Pull request merged**